.github/workflows/docker_image_build.yaml

```
@@ -0,0 +1,22 @@
+name: Building the Docker image containing the basic tools for building ParallelZone
+on:
+  push:
+    path:
+      Docker/**
```

```suggestion
      Docker/ParallelZone.Dockerfile
```
If we end up with multiple Docker files, we don't want to rebuild the PZ image when the other Docker files change.

---

True. Thanks for your change.